maps/_us-healthcare-system-capacity/scripts/cibuild

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${COVID19_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Build the site!
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        mkdir -p build
+        mkdir -p data/config
+
+        # Copy processed configuration files
+        cp "${DIR}/../../../data/processed/ccm_county_breaks.json" data/config/
+        cp "${DIR}/../../../data/processed/ccm_state_breaks.json" data/config/
+        cp "${DIR}/../../../data/processed/ccm_hrr_breaks.json" data/config/
+        cp "${DIR}/../../../data/processed/ccm_facility_breaks.json" data/config/
+
+        # Copy published GeoJSON files
+        cp "${DIR}/../../../data/published/us_healthcare_capacity-state-CovidCareMap.geojson" build/
+        cp "${DIR}/../../../data/published/us_healthcare_capacity-county-CovidCareMap.geojson" build/
+        cp "${DIR}/../../../data/published/us_healthcare_capacity-hrr-CovidCareMap.geojson" build/
+        cp "${DIR}/../../../data/published/us_healthcare_capacity-facility-CovidCareMap.geojson" build/
+
+        # Replace all instances of NaN with null
+        sed -i.bak 's/NaN/null/' build/*.geojson
+
+        # Install tippecanoe
+        wget -qO- https://github.com/mapbox/tippecanoe/archive/1.32.5.tar.gz |
```

Comment:
Dockerizing the running of the CI scripts would allow this to be installed once on docker build instead of each time the script is run.

Perhaps using the https://github.com/netlify/build-image container as a base and setting environment variables that would be checked in the cibuild scripts to skip install steps.